server/controllers/signupControllers/signup.js

```diff
@@ -0,0 +1,27 @@
+const { createCookies, createSession } = require('../../utils');
```

Comment:
Is createSession exported from utils?

Reply:
yes, but it is in another branch and we didn't merge it yet!

Reply:
fix it in this branch, as these changes are for this branch!